types/config.d.ts

```diff
@@ -62,10 +62,10 @@ declare module Config {
    * This information is typically set by the CLI or other channel integrations.
    */
   interface FRContext {
-    gatherMode?: Gatherer.GatherMode;
```

Comment:
drive-by: this shouldn't be exposed on the API. The first thing we do with it is force override with the correct value from whichever runner it's a parameter for.